src/domain/Day.ts

```diff
@@ -30,9 +30,9 @@ export function getToday(block: ethereum.Block): Day {
 
 // if the difference between days since nonzerobalance
 // and today's timestamp are greater than or equal to sixty days in seconds
-// if kovan, we want to make rewards eligible after 1 day
-export function rewardsAreEligible(today: BigInt, nonZeroSince: BigInt): boolean {
-  return dataSource.network() == 'mainnet'
-    ? nonZeroSince.minus(BigInt.fromI32(secondsInSixtyDays)) >= today
-    : nonZeroSince.minus(BigInt.fromI32(secondsInDay)) >= today
+// if kovan, we want to make rewards claimable after 1 day
+export function rewardsAreClaimable(today: BigInt, nonZeroSince: BigInt | null): boolean {
+  return nonZeroSince != null && dataSource.network() == 'mainnet'
+    ? today.minus(<BigInt>nonZeroSince).ge(BigInt.fromI32(secondsInSixtyDays))
+    : today.minus(<BigInt>nonZeroSince).ge(BigInt.fromI32(secondsInDay))
```

Comment:
There are parentheses missing here: 

```suggestion
  return nonZeroSince != null && (dataSource.network() == 'mainnet'
    ? today.minus(<BigInt>nonZeroSince).ge(BigInt.fromI32(secondsInSixtyDays))
    : today.minus(<BigInt>nonZeroSince).ge(BigInt.fromI32(secondsInDay)))
```